draft-ietf-taps-transport-security.md

```diff
@@ -44,6 +44,14 @@ author:
     city: Cupertino, California 95014
     country: United States of America
     email: cawood@apple.com
+  -
+    ins: T. Enghardt
```

Comment:
nit: We keep these in alphabetical order -- can you please insert your name higher up in the list?

Reply:
As an academic, it feels weird to see my name first in that list... but okay ;)

Reply:
If you want, I can make myself editor, and then list everyone else after?

Reply:
That would work for me, as long as everyone else agrees, too.

Reply:
I'm happy either way...

@theri take the credit ;-)

Reply:
@theri Can you please update, either with the editor suggestion or your name first? Thanks!

Reply:
Done, with the editor suggestion.